bound/kt/src/main/kotlin/web5/sdk/crypto/keys/KeyManager.kt

```diff
@@ -0,0 +1,32 @@
+package web5.sdk.crypto.keys
+
+import web5.sdk.crypto.signers.Signer
+
+/**
+ * An interface representing a key manager for cryptographic operations.
+ */
+interface KeyManager {
```

Comment:
this isn't in the APID 0.1.0 FWIW but will probably come in the next version when we add bearer DIDs. it's no problem to include it here, but just calling it out in case you weren't aware (& also it may change when we add it in the next version of the APID). it's legal, according to RFC-0001 for implementations to go above the APID, so this is allowed per the discretion of each implementor

> The APID is asserted to act as a floor, or a baseline, of feature support — each implementation may choose to expose publicly facing APIs which are beyond the scope of the APID.